roles/pyxis/tasks/main.yml

```diff
@@ -8,10 +8,9 @@
     preflight_output: "{{ lookup('file', preflight_results_file) | from_json }}"
     preflight_artifact: "{{ lookup('file', preflight_log_file) | to_json }}"
     pyxis_apikey: "{{ lookup('file', pyxis_apikey_path) }}"
+    preflight_release: "{{ preflight_output.test_library.version.split('+')[0] }}"
```

Comment:
From the last failure I believe we need to define `preflight_release:` fact separately after this task or maybe move the fact `preflight_output:` as a var in this task?